pangeo_forge_recipes/writers.py

```diff
@@ -112,13 +112,15 @@ def write_combined_reference(
 ) -> FSSpecTarget:
     """Write a kerchunk combined references object to file."""
 
-    import ujson  # type: ignore
+    from fsspec.implementations.reference import ReferenceFileSystem
 
     file_ext = os.path.splitext(output_file_name)[-1]
-
     outpath = full_target._full_path(output_file_name)
+    remote_protocol = _select_single_protocol(full_target)
```

Comment:
```suggestion
    target_protocol = _select_single_protocol(full_target)
```

See other comment re: difference between `remote_protocol` and `target_protocol`. IIUC, we have these backwards at the moment.

Reply:
So where/how can the remote protocol be inferred? Or should it be passed explicitly? 

Also, should all the the kwargs that we pass to `MultiZarrToZarr` and `ReferenceFileSystem` be packaged as mzz_kwargs and unpacked or separated?